LFS171x/composer-material/sections/section_3.md

```diff
@@ -0,0 +1,147 @@
+# Hyperledger Composer Architecture
+
+<!-- TAB 1 -->
+## Hyperledger Composer Key Components
+  
+![Hyperledger Composer Key components overview](resources/img_03-01.png)
+
+<!-- TAB 2 -->
+## Business Network
+
+![Business Network](resources/img_03-02.png)
+
+A **Business Network** includes:
+- *Modeling language files (`models/.cto`)* to define models for Participants, Assets, Transactions and Events.
+- *Transaction Logic (`lib/.js`)* to implement the logic of the transactions defined
+- *Query file (`queries.qry`)* to design and enable complex queries on the Blockchain Data
+- *Access Control File (`permissions.acl`)* to control visibility and actions on resources
+
+![Business Network Folder](resources/img_03-03.png)
+
+<!-- TAB 3 -->
+## Business Network - Modeling language
+
+![Icon Participants](resources/icon_participants.png)
+![Icon Assets](resources/icon_assets.png)
+![Icon Transactions](resources/icon_transactions.png)
+![Icon Events](resources/icon_events.png)
+
+Hyperledger Composer includes an object-oriented modeling language that is used to define the domain model for a business network definition.
```

Comment:
Would it be worth including a link to the modeling language section of the documentation (https://hyperledger.github.io/composer/latest/reference/cto_language)?